lib/gitlab_merge_request_hook.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'gitlab_init'
+
+class GitlabMergeRequestHook
+  attr_reader :config, :repo_path, :changes
+
+  def initialize(repo_path, config, changes)
+    @branch    = branch_name(changes)
+    @config    = config
+    @repo_path = repo_path
+  end
+
+  def exec
+    return if @branch.nil? || @branch == "master"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.